skimage/measure/block.py

```diff
@@ -18,6 +18,8 @@ def block_reduce(image, block_size, func=np.sum, cval=0):
     cval : float
         Constant padding value if image is not perfectly divisible by the
         block size.
+    axis : int
+        Specify the axis on which func will be called. Default is -1
```

Comment:
I would avoid saying "Default is" as people can infer this from the signature. See for example: http://scikit-image.org/docs/stable/api/skimage.graph.html#skimage.graph.MCP

This is useful if `None` is in the signature, but later down, the code would translate `None` into a valid option.

Reply:
Thanks for the feedback. I agree! I made some modifications on the code and now the default value is None, therefore I applied what you pointed out here. I could still be improvable, let me know